internal/pkg/fixtures/fixtures.go

```diff
@@ -51,8 +51,9 @@ type ConfigRow struct {
 }
 
 type StateFile struct {
-	AllBranchesConfigs []string       `json:"allBranchesConfigs" validate:"required"`
-	Branches           []*BranchState `json:"branches" validate:"required"`
+	AllBranchesConfigs []string          `json:"allBranchesConfigs" validate:"required"`
+	Branches           []*BranchState    `json:"branches" validate:"required"`
+	Envs               map[string]string `json:"envs,omitempty"` // additional ENVs
```

Comment:
Tato struktura reprezentuje `initial-state.json` z funkcionalnych testov.

Ak chcem otestovat `variables`, tak musim v teste nahradit `variables_id` v konfiguracii, ktora variables pouziva.

Su tam 2 problemy:
- jedna sa o konfiguracie, tj. vytvaraju sa paralelne, a nepoznal som ich ID
- testovaci config je definovany JSON suborom, napr. https://github.com/keboola/keboola-as-code/blob/main/internal/pkg/fixtures/configs/python-transformation-shared-code.json ... ale tento config sa prida v `initial-state.json` do nejakej lubovolnej `branch`-e
- no a ID branche, alebo configu v branchi obsahuje jej nazov: https://github.com/keboola/keboola-as-code/blob/35cf13f34d8eb61038fb88e0134c43cf2ef0b7bc/test/pull/out/.keboola/manifest.json#L119
- tj. dopredu nevieme, aky bude nazov ENV premennej

Druhy problem som vyriesil tak, ze je mozne definovat vlastne `Envs` v `initial-state.json`.

Lepsie to vysvetlim na praktickej ukaze v dalsom PR, tu sa to iba pripravuje, bez zmeny testov.

Reply:
Dalsi PR: https://github.com/keboola/keboola-as-code/pull/257